.drone.star

```diff
@@ -534,6 +534,38 @@ def changelog(ctx):
 
 	return pipelines
 
+def unitTests():
+	return [{
+		'kind': 'pipeline',
+		'type': 'docker',
+		'name': 'unit-tests',
+		'steps': [
+			{
+				'name': 'install-deps',
+				'image': 'owncloudci/nodejs:12',
+				'pull': 'always',
+				'commands': [
+					'yarn install',
```

Comment:
can we run the `installNPM()` instead of making an own step here?

Reply:
Done